src/main/java/edu/cornell/kfs/fp/document/service/CuDisbursementVoucherTaxService.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.kfs.fp.document.service;
+
+public interface CuDisbursementVoucherTaxService {
+	
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId);
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
```

Comment:
Sandy, would it make more sense to rename them to: 
"isForeignVendorAndTaxReviewRequired" and "isForeignVendorAndTaxReviewNotRequired"? Thank you.